README.md

```diff
@@ -49,6 +49,14 @@ For alpha versions of Volto:
 USE_VOLTO_ALPHA=1 pipx run cookiecutter gh:collective/cookiecutter-plone-starter
 ```
 
+
+### Restarting cookiecutter with previous answers
+
+The cookiecutter template will ask a quite some questions.
```

Comment:
```suggestion
The cookiecutter will ask a lot of questions.
You can use some of its options to avoid repeatedly entering the same values.


#### `--no-input`

You can use the [`--no-input`](https://cookiecutter.readthedocs.io/en/latest/cli_options.html#cmdoption-cookiecutter-no-input) option to make the cookiecutter not prompt for parameters and only use `cookiecutter.json` file content.


#### `--replay` and `--replay-file`

```